src/report/business-reports/aging.scm

```diff
@@ -632,8 +789,43 @@ totals to report currency.")
 						(cdr company-list-entry))))
 			       (owner (company-get-owner-obj
 				       (cdr company-list-entry)))
-			       (company-name (gncOwnerGetName owner)))
-
+			       (company-name (gncOwnerGetName owner))
+			       (addr (get-addr owner disp-addr-source))
+			       (addr-name  (gncAddressGetName  addr))
+			       (addr-addr1 (gncAddressGetAddr1 addr))
+			       (addr-addr2 (gncAddressGetAddr2 addr))
+			       (addr-addr3 (gncAddressGetAddr3 addr))
+			       (addr-addr4 (gncAddressGetAddr4 addr))
+			       (addr-phone (gncAddressGetPhone addr))
+			       (addr-fax   (gncAddressGetFax   addr))
+			       (addr-email (gncAddressGetEmail addr))
+			       (company-active (if (gncOwnerGetActive owner)
+			         (_ "Y") (_ "N")))
+			       (opt-fld-list '())
+			      )
+;;            (gnc:debug "aging-renderer: disp-addr-source=" disp-addr-source
+;;              " owner=" owner
+;;              " gncOwnerGetID="  (gncOwnerGetID owner)	;; cust no
+;;              " gncCustomerGetShipAddr="                  
+;;                (gncCustomerGetShipAddr (gncOwnerGetCustomer owner)))
+			  (if disp-addr-name
+			    (set! opt-fld-list (append opt-fld-list (list addr-name))))
+			  (if disp-addr1
+                (set! opt-fld-list (append opt-fld-list (list addr-addr1))))
+			  (if disp-addr2
+                (set! opt-fld-list (append opt-fld-list (list addr-addr2))))
+			  (if disp-addr3
+                (set! opt-fld-list (append opt-fld-list (list addr-addr3))))
+			  (if disp-addr4
+                (set! opt-fld-list (append opt-fld-list (list addr-addr4))))
+			  (if disp-addr-phone
+                (set! opt-fld-list (append opt-fld-list (list addr-phone))))
+			  (if disp-addr-fax
+                (set! opt-fld-list (append opt-fld-list (list addr-fax))))
+			  (if disp-addr-email
+                (set! opt-fld-list (append opt-fld-list (list addr-email))))
+			  (if disp-active
+                (set! opt-fld-list (append opt-fld-list (list company-active))))
```

Comment:
The indentation here gets a bit weird due to a mixture of tabs and spaces. Can you clean that up ?